compiler/src/iree/compiler/Dialect/HAL/Target/LLVMCPU/LLVMCPUTarget.cpp

```diff
@@ -444,17 +445,20 @@ class LLVMCPUTargetBackend final : public TargetBackend {
              << targetTriple.str() << "'";
     }
 
-    // Link musl last and pull in all of it - this is sad but LLVM will take IR
-    // intrinsics and generate calls out to libc during code generation and we
-    // have no control over that - if we don't provide the symbols here then
-    // linking with ld will fail.
-    if (failed(linkBitcodeModule(
-            variantOp.getLoc(), moduleLinker, llvm::Linker::OverrideFromSrc,
-            *targetMachine, "libmusl",
-            loadMuslBitcode(targetMachine.get(), context)))) {
-      return mlir::emitError(variantOp.getLoc())
-             << "failed linking in builtin library for target triple '"
-             << targetTriple.str() << "'";
+    if (options_.linkEmbedded) {
+      // Link musl last and pull in all of it - this is sad but LLVM will take
+      // IR
+      // intrinsics and generate calls out to libc during code generation and we
+      // have no control over that - if we don't provide the symbols here then
+      // linking with ld will fail.
```

Comment:
Can you re-flow this comment block? (nesting it one level pushed a line over the limit and clang-format doesn't re-flow on its own)
```suggestion
      // Link musl last and pull in all of it - this is sad but LLVM will take
      // IR intrinsics and generate calls out to libc during code generation
      // and we have no control over that - if we don't provide the symbols
      // here then linking with ld will fail.
```